src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+  static removeDead() {
+    Animal.alive = Animal.alive.reduce((acc, animal) => {
+      if (animal.health > 0) {
+        acc.push(animal);
+      }
+
+      return acc;
+    }, []);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    if (animal instanceof Carnivore || animal.hidden) {
+      return 'The bite method does not work.';
```

Comment:
you no need to return any string
you can use only 'return'-statement or make invert condition and decrease here animal health